src/file_operations.js

```diff
@@ -69,35 +95,19 @@ const scanCourses = async (inputPath, outputPath) => {
     return
   }
 
-  // populate the course uid mapping
```

Comment:
Note: this is made obsolete because we are including course uid => path mapping within `pathLookup`, so this code and all uses of `courseUidsLookup` were removed

Reply:
Thanks for clarifying, I had thought that was the case but apparently didn't look closely enough.